configdialog.cpp

```diff
@@ -92,8 +111,23 @@ void ConfigDialog::refresh() {
 		SendMessage(slider, TBM_SETPOS, TRUE, encodef(m_current_config.at(entry.first)));
 	}
 
+	if (m_current_config.at(YonkPattern) < 0 || m_current_config.at(YonkPattern) >= _PATTERN_COUNT)
```

Comment:
Something about all of this is hurting my 3 am brain and I don't know what to make of it. I'll come back to this.